.scalafmt.conf

```diff
@@ -0,0 +1,3 @@
+version = 3.2.1
```

Comment:
can we locate this file under `build-support/jvm` instead of putting under repo root ?

Reply:
I don't think so... it's similar to `pyproject.toml` in that regard: tools expect it at the root of the repo (or at least "above the code being checked"). While we could special case it here, it's easiest to stick with the convention.